TestAssets/TestProjects/MSBuildTestApp/project.json

```diff
@@ -11,6 +11,10 @@
   "runtimes": {
     "win7-x64": {},
     "win7-x86": {},
+    "win81-x64": {},
```

Comment:
@rrelyea @jasonmalinowski @Pilchie - Can you guys answer this?  Should we expect the nuget targets in MSBuild to understand RID-fallback?  We can log an issue if this is expected to work.